File: AtlasWeb/Core/Modules/Tibbi_Surec/Fizyoterapi_Modulu/ScoliosisAssessmentFormViewModel.ts
//$05DF2D9F
import { BaseViewModel } from "NebulaClient/Model/BaseViewModel";
import { ScoliosisAssessmentForm } from "NebulaClient/Model/AtlasClientModel";
import { Type } from 'NebulaClient/ClassTransformer';

export class ScoliosisAssessmentFormViewModel extends BaseViewModel {
    protected __type__: string = 'Core.Models.ScoliosisAssessmentFormViewModel, Core';
    @Type(() => ScoliosisAssessmentForm)
    public _ScoliosisAssessmentForm: ScoliosisAssessmentForm = new ScoliosisAssessmentForm();
}

File: AtlasWeb/Core/Modules/Tibbi_Surec/Fizyoterapi_Modulu/ScoliosisAssessmentFormViewModel.cs
//$6AC9966C
using System;
using System.Linq;
using Core.Models;
using Infrastructure.Filters;
using Infrastructure.Models;
using TTInstanceManagement;
using TTObjectClasses;
using TTDefinitionManagement;
using TTUtils;
using Core.Controllers;

namespace Core.Controllers
{
    public partial class ScoliosisAssessmentFormServiceController
    {
    }
}

namespace Core.Models
{
    public partial class ScoliosisAssessmentFormViewModel : BaseAdditionalInfoFormViewModel
    {
        public override BaseAdditionalInfo AddViewModelToContext(TTObjectContext objectContext)
        {
            var scoliosisAssessmentFormService = new ScoliosisAssessmentFormServiceController();
            scoliosisAssessmentFormService.ScoliosisAssessmentFormInternal(this, objectContext);
            return (ScoliosisAssessmentForm)objectContext.GetObject(this._ScoliosisAssessmentForm.ObjectID, "ScoliosisAssessmentForm");
        }
    }
}
